docs-markdown/src/test/suite/index-coverage.ts

```diff
@@ -61,18 +62,18 @@ export function run(): Promise<void> {
 					if (failures > 0) {
 						e(new Error(`${failures} tests failed.`));
 					} else {
+						nyc = setupNyc();
```

Comment:
Running this again here correctly wraps the tested code files with nyc magic, and then spits out an _accurate_ report.